ancpbids/pybids_compat.py

```diff
@@ -252,6 +268,17 @@ def get_dataset(self) -> object:
         """
         return self.dataset
 
+    def add_derivatives(self, path):
```

Comment:
there is an example here:
https://github.com/ANCPLabOldenburg/ancp-bids-apps/blob/main/ancpbidsapps/analysis/nilearn_first_level.py#L66

use schema API to create new schema objects

Reply:
Feel free to issue a PR to this PR in whatever way you think is most performant given the way you organize data within this architecture. This is not meant to be just an effort dictated by my skill and coding ability, but a collaborative one :smile: 

Check out https://github.com/bids-standard/pybids/pull/851 to see the workflow I've been going through so you can start testing pieces, too.